frontend/src/components/LinkInformations/RedeemInformation.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <div class="redeem-information">
+    <b-jumbotron bg-variant="muted" text-variant="dark" border-variant="info">
+      <h1>
+        {{ user.firstName }}
+        {{ $t('transaction-link.send_you') }} {{ amount | GDD }}
+      </h1>
+      <b>{{ memo }}</b>
+    </b-jumbotron>
+  </div>
+</template>
+<script>
+export default {
+  name: 'RedeemInformation',
+  props: {
+    user: { type: Object, required: true },
```

Comment:
just use `name` type String - you only seem to use the `firstName` field on user

Reply:
es wird firstName übermittelt